go/teams/loader.go

```diff
@@ -823,7 +823,7 @@ func (l *TeamLoader) userPreload(ctx context.Context, links []*ChainLinkUnpacked
 			for uid := range uidSet {
 				wg.Add(1)
 				go func(uid keybase1.UID) {
-					_, _, err := l.G().GetUPAKLoader().LoadV2(
+					_, err := l.G().GetUPAKLoader().LoadLite(
```

Comment:
Only preload lite chains. Not sure if we want admins to preload the entire chain, but then I guess we don't know if we're an admin before loading.

Reply:
do we actually want to start doing this?